tracee/container_tracer.py

```diff
@@ -833,6 +833,8 @@ def print_event(self, cpu, data, size):
 
         args = list()
 
+        eventfunc = "dummy"
```

Comment:
What's this for?

Reply:
As eventfunc is set only for the events that have a different function name than the event name (e.g. lstat), and not for all the other events, it is initialized to this value here in order to emphasis the fact that it shouldn't be found in events_to_trace later on for events which were not chosen by the user (line 1065 in this commit)